src/share/poudriere/include/fs.sh

```diff
@@ -349,7 +349,7 @@ destroyfs() {
 	local type="$2"
 	local fs
 
-	umountfs ${mnt} 1
+	umountfs ${mnt}
 	if [ ${TMPFS_ALL} -eq 1 ]; then
 		if [ -d "${mnt}" ]; then
 			if ! umount ${UMOUNT_NONBUSY} "${mnt}" 2>/dev/null; then
```

Comment:
This unmounts the main mount for tmpfs.
Line 362 unmounts the main mount for zfs.
It seems that `NO_ZFS` may not unmount the main mount. Is that the configuration you are using?